wasabi.gemspec

```
@@ -12,14 +12,13 @@ Gem::Specification.new do |s|
   s.description = s.summary
   s.required_ruby_version = '>= 1.9.2'
 
-  s.rubyforge_project = s.name
   s.license = 'MIT'
 
   s.add_dependency "httpi",    "~> 2.0"
   s.add_dependency "nokogiri", ">= 1.4.2"
 
-  s.add_development_dependency "rake",  "~> 0.9"
-  s.add_development_dependency "rspec", "~> 2.14"
+  s.add_development_dependency "rake",  "~> 13.0.1"
+  s.add_development_dependency "rspec", "~> 3.7.0"
```

```suggestion
  s.add_development_dependency "rake",  "~> 13.0"
  s.add_development_dependency "rspec", "~> 3.7"
```

Is there a reason for being so precise?